src/driver/kvm.rs

```diff
@@ -205,95 +265,9 @@ impl<T: KVMIntrospectable> Drop for Kvm<T> {
             self.kvmi
                 .control_events(vcpu, KVMiInterceptType::Cr, false)
                 .unwrap();
-        }
-    }
-}
-
-#[cfg(test)]
```

Comment:
Why are you deleting the tests? 😱

Reply:
Sir
The tests are present in the unit_tests branch

Reply:
Yes, your new tests. But those are the tests that are already present in the master branch. You should not delete them.